src/canvas-view/canvas/node-toolbar.component.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('nodeToolbar', {
+      templateUrl: 'canvas-view/canvas/node-toolbar.html',
+      bindings: {
+        node: '=',
+        nodeActions: '=',
+        nodeClickHandler: '=',
+        nodeCloseHandler: '='
+      },
```

Comment:
Same comment regarding using one-way binds instead of two-way on the ```-Handlers``` using ```<```, do we need the handlers passed back out?

Reply:
From my knowledge, I don't believe the handlers need to be passed back out. Can change these to one-way binds as well.